### ISSUE TITLE
Unhandled Runtime Error when using next 13.1.1

### ISSUE THREAD

**Issue by michalstrzelecki** [aralroca/next-translate#961]
**What version of this package are you using?**
`1.6.0`

**What operating system, Node.js, and npm version?**
```
    Operating System:
      Platform: linux
      Arch: x64
      Version: #62~20.04.1-Ubuntu SMP Tue Nov 22 21:24:20 UTC 2022
    Binaries:
      Node: 18.12.0
      npm: 8.19.2
      Yarn: N/A
      pnpm: N/A
    Relevant packages:
      next: 13.1.1
      eslint-config-next: N/A
      react: 18.2.0
      react-dom: 18.2.0
```

**What happened?**
I want you to reperoduce the error. To reproduce the error please clone the repo [https://github.com/michalstrzelecki/next-translate-n1311](https://github.com/michalstrzelecki/next-translate-n1311) and run `npm run dev`. Then create the `src` folder in the root of the project and simply move the `pages` folder there. Restart the dev server. The error will appear.
![obraz](https://user-images.githubusercontent.com/3853055/210546357-487959a4-f458-4730-aef2-bd3d8155ceeb.png)

I find it the only way to reproduce the error this way. In my work project I do not move the files all around and the error is there already on the next 13.1.1. 

I also realized that removing all dynamic imports from `_app` file, when I am using next 13.1.1, solves the issue.

**What did you expect to happen?**
I expect to use this package with with next 13.1.1.

**Are you willing to submit a pull request to fix this bug?**
I do not know what to do next.

**Comment by aralroca** [comment]
I can't reproduce it, looks that works fine for me

<img width="1488" alt="Screenshot 2023-01-09 at 15 06 11" src="https://user-images.githubusercontent.com/13313058/211326661-9e2cab09-db35-48a1-bfe1-70657f87505f.png">

Node v16.17.0 
NPM 8.15.0

**Comment by michalstrzelecki** [comment]
Could you use node 18.12.0? I just want to update to the nextjs version - 13.1.5 - and I am still getting the same error.

**Comment by michalstrzelecki** [comment]
I have checked it using node 16.17.0 and I get this strange error also.

**Comment by michalstrzelecki** [comment]
I have just updated the repo, please try again.

**Comment by aralroca** [comment]
Sorry, @michalstrzelecki, I can't reproduce it. After creating the `src` folder and moving `pages` inside, `npm run dev` works fine 🤔

Anyway, it doesn't seem to be a bug related to Next-translate, maybe it has more to do with Next.js? Have you tried [this canary](https://github.com/vercel/next.js/releases/tag/v13.1.7-canary.11) version of Next.js to see if it doesn't happen now? In this pre-release there is a fix in the dynamic import.

**Comment by michalstrzelecki** [comment]
@aralroca thanks for interests. It has been fixed in the `v13.1.7-canary.9` release of nextjs.
